src/test/visit_test_main.py

```diff
@@ -941,6 +945,14 @@ def Test(case_name, altSWA=0, alreadySaved=0):
             g = GetGlobalAttributes()
             win = g.windows[g.activeWindow]
             ResizeWindow(win, width, height)
+            ## the following can be used to save a JSON file and compare
```

Comment:
i used this for debugging -- kept it here b/c it could be useful in the future, but not planning to re-enable. 
If we want to augment what is saved, we might be able to do that in the larger json results.